recipes/lora_finetune_distributed.py

```diff
@@ -60,7 +60,7 @@ class LoRAFinetuneRecipeDistributed(FTRecipeInterface):
     The following configs can be used to run this recipe:
         >>> tune ls
         RECIPE                         CONFIG
-        lora_finetune_distributed      lora_finetune_distributed
```

Comment:
Have you actually ran this command? I think it goes on multiple lines?

Reply:
yup!
![image](https://github.com/pytorch/torchtune/assets/47255723/000ca39d-19b4-477e-acdf-20228f734112)